core/generator.js

```diff
@@ -402,32 +402,50 @@ Blockly.Generator.prototype.definitions_;
 Blockly.Generator.prototype.functionNames_;
 
 /**
- * A database of variable names.
+ * A database of variable and procedure names.
  * @type {Blockly.Names}
  * @protected
  */
-Blockly.Generator.prototype.variableDB_;
+Blockly.Generator.prototype.nameDB_;
 
 /**
- * Define a function to be included in the generated code.
+ * @deprecated 'variableDB_' was renamed to 'nameDB_' (May 2021).
+ */
+Object.defineProperty(Blockly.Generator.prototype, 'variableDB_', {
+  get: function() {
+    console.warn('Deprecated use of "variableDB_"; change to "nameDB_"');
+    return this.nameDB_;
+  },
+  set: function(x) {
+    console.warn('Deprecated use of "variableDB_"; change to "nameDB_"');
+    this.nameDB_ = x;
+  }
+});
```

Comment:
This seems like a nice way to deprecate something! I'll have to keep this in mind in the future.